tools/copyexp/copyexp.cpp

```diff
@@ -226,7 +226,7 @@ int copyExpanded(const char *from, const char *to, bool stats)
 }
 
 
-void copyCompress(const char *from, const char *to, size32_t rowsize, bool fast, bool flzstrm, bool lz4, bool lz4strm, bool stats)
+void copyCompress(const char *from, const char *to, size32_t rowsize, bool lzw, bool fast, bool flzstrm, bool lz4, bool lz4strm, bool stats)
```

Comment:
This seems like a very weird prototype!

Reply:
I agree, I will change it to bit-value or string if you want.

Reply:
does anything use the tool copyexp?
This is not related to default compression format of files created by the engines.
I'd tackle separately from this issue.